include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -2,15 +2,73 @@
 
 #include<iostream>
 #include<pjsua2.hpp>
+#include<unordered_map>
+#include <string>
 
 #include"Softphone.hpp"
+#include"SoftphoneArguments.hpp"
 
 class SoftphoneManager
 {
 public:
-    SoftphoneManager();
-    ~SoftphoneManager();
+    SoftphoneManager(int number_of_softphones, int port, std::string domain):
+        _port(port), _domain(std::move(domain))
+    {
+        pjLibraryInit();
+    }
+
+    ~SoftphoneManager()
+    {
+        _endpoint.libDestroy();
+    }
+
+    void pjLibraryInit()
+    {
+        try
+        {
+            //Create library
+            _endpoint.libCreate();
+
+            // Init library
+            pj::EpConfig ep_cfg;
+            ep_cfg.logConfig.level = 1;
+            _endpoint.libInit( ep_cfg );
+
+            // Transport
+            pj::TransportConfig tcfg;
+            tcfg.port = _port;
+            _endpoint.transportCreate(PJSIP_TRANSPORT_UDP, tcfg);
+
+            // Start library
+            _endpoint.libStart();
+            std::cout << "*** PJSUA2 STARTED ***" << std::endl;
+        }
+        catch(pj::Error & err)
+        {
+            std::cerr << "Exception: " << err.info() << std::endl;
+        }
+    }
+
+    void createSoftphone(int number_of_softphones)
```

Comment:
Rename to `registerSoftphones` and the parameter to `amount`

Reply:
done